### PR TITLE
Keybinding: Error Message Localization

### LINKED ISSUE #12878: Information label "unrecognized key" is not localized in HotKey Binding
### Bug Description:

image

https://github.com/eclipse-theia/theia/blob/5d7a655eaeb42809be8128ec4e92fa8f34d6d7dc/packages/core/src/common/keys.ts#L291

### Additional Information

I will be able to fix this issue a little bit later by myself, if no one else fixes it soon :)

- Operating System: any
- Theia Version: the latest

### PR DESCRIPTION
**Pull request by FernandoAscencio**
#### What it does
Closes #12878

#### How to test
Same steps as in #12878

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/core/src/common/keys.ts [hunk 1]
```diff
@@ -252,7 +253,7 @@ export class KeyCode {
         }
         /* If duplicates i.e ctrl+ctrl+a or alt+alt+b or b+alt+b it is invalid */
         if (keys.length !== new Set(keys).size) {
-            throw new Error(`Can't parse keybinding ${keybinding} Duplicate modifiers`);
+            throw new Error(nls.localize('theia/core/keybinding/error.duplicateModifier', "Can't parse keybinding {0} Duplicate modifiers", keybinding));
```
Comment (vince-fugnitto) [inline packages/core/src/common/keys.ts:256]: I don't believe we use `.` in our keys at the moment, so I'd update for consistency:

```suggestion
            throw new Error(nls.localize('theia/core/keybinding/duplicateModifierError', "Can't parse keybinding {0} Duplicate modifiers", keybinding));
```

File: packages/core/src/common/keys.ts [hunk 2]
```diff
@@ -266,7 +267,7 @@ export class KeyCode {
                 if (isOSX) {
                     schema.meta = true;
                 } else {
-                    throw new Error(`Can't parse keybinding ${keybinding} meta is for OSX only`);
+                    throw new Error(nls.localize('theia/core/keybinding/error.OsxOnly', "Can't parse keybinding {0} meta is for OSX only", keybinding));
```
Comment (vince-fugnitto) [inline packages/core/src/common/keys.ts:270]:
```suggestion
                    throw new Error(nls.localize('theia/core/keybinding/metaError', "Can't parse keybinding {0} meta is for OSX only", keybinding));
```

File: packages/core/src/common/keys.ts [hunk 3]
```diff
@@ -288,7 +289,7 @@ export class KeyCode {
                     schema.key = key;
                 }
             } else {
-                throw new Error(`Unrecognized key '${keyString}' in '${keybinding}'`);
+                throw new Error(nls.localize('theia/core/keybinding/error.unrecognizedKey', 'Unrecognized key {0} in {1}', keyString, keybinding));
```
Comment (vince-fugnitto) [inline packages/core/src/common/keys.ts:292]:
```suggestion
                throw new Error(nls.localize('theia/core/keybinding/unrecognizedKeyError', 'Unrecognized key {0} in {1}', keyString, keybinding));
```

**Comment by FernandoAscencio** [comment]
@vince-fugnitto 
Makes sense. The changes have been comited.

**Review by msujew — APPROVED** [review]
Looks good to me 👍

**Pull request merged**